### DISCUSSION TITLE
Searching for Creep Models in Moose

### DISCUSSION THREAD

**Discussion by hardud** [idaholab/moose#27476] [Q&A Modules: Solid mechanics]
Hi,

Is it possible to look at what all Creep models moose already has? I found Power Law Creep Stress Update, but if I want to apply another creep model, where can I look at all the options?

Thanks

**Comment by GiudGiud** [comment]
Hello

The index of all objects available in MOOSE is at this page
https://mooseframework.inl.gov/source/index.html
You can search for creep there
Alternatively, all materials here:
https://mooseframework.inl.gov/syntax/Materials/index.html

Guillaume

**Reply by hardud** [reply]
Thanks Dr. Guillaume,

Under these I could only find two materials that computes Creep: Power Law Creep and Hill Creep. Is it possible to add other common creep models for eg. Norton-Bailey Creep Law or Garofalo Creep Law? 

Best,
Harsh

**Reply by GiudGiud** [reply]
Hello

It certainly is. Are you looking into implementing them yourself?

Guillaume

**Reply by hardud** [reply]
Yes, I would need to use some of the creep models for my research that are not available in moose. I can model them and let you know. 

Harsh

**Reply by GiudGiud** [reply]
sounds good. @bwspenc @gambka any of these two models you would be interested in seeing contributed back to MOOSE?
